src/zenml/integrations/kubernetes/orchestrators/manifest_utils.py

```diff
@@ -74,9 +77,61 @@ def build_pod_manifest(
     if service_account_name is not None:
         spec = cast(Dict[str, Any], manifest["spec"])  # mypy stupid
         spec["serviceAccountName"] = service_account_name
+
+    if settings is not None:
+        spec = cast(Dict[str, Any], manifest["spec"])
+        spec.update(add_pod_settings(settings))
     return manifest
 
 
+def add_pod_settings(settings):
```

Comment:
these three functions need docstrings and type annotations

Reply:
I just enabled workflows for your PR @wefner, these will give you more details on what's missing, see below :)

Reply:
To fix the spell-check issues, you will need to add `tolerations` into `.pyspelling-ignore-words`.

The other issues can be solved by wrapping class names in backticks (`), e.g. `KubernetesOrchestratorSettings`